src/api/schemas/events.js

```diff
@@ -1,8 +1,14 @@
 import { schema } from 'normalizr';
+import { repoSchema } from './repos';
 
 export const eventSchema = new schema.Entity(
   'events',
-  {},
+  {
+    repo: repoSchema,
+    payload: {
+      forkee: repoSchema,
+    },
+  },
```

Comment:
This is how we hint Normalizr about a nested schema.

The original struct event was:
```
  {
    "id": "7584480420",
    "type": "WatchEvent",
    "actor": {
      "id": 32770098,
      "login": "machour2",
      "display_login": "machour2",
      "gravatar_id": "",
      "url": "https://api.github.com/users/machour2",
      "avatar_url": "https://avatars.githubusercontent.com/u/32770098?"
    },
    "repo": {
      "id": 86202845,
      "name": "gitpoint/git-point",
      "url": "https://api.github.com/repos/gitpoint/git-point"
    },
    "payload": {
      "action": "started"
    },
    "public": true,
    "created_at": "2018-04-25T12:13:40Z",
    "org": {
      "id": 30082377,
      "login": "gitpoint",
      "gravatar_id": "",
      "url": "https://api.github.com/orgs/gitpoint",
      "avatar_url": "https://avatars.githubusercontent.com/u/30082377?"
    }
  },
```

This is then normalized to:

<img width="292" alt="screen shot 2018-04-25 at 2 00 24 pm" src="https://user-images.githubusercontent.com/304450/39247383-2f0d848a-4891-11e8-9f15-0fb111188cc6.png">

`repo` is now the repoId instead of the repo struct, pointing to our minimalistic v3 representation in state.entities.repos:
<img width="459" alt="screen shot 2018-04-25 at 1 15 26 pm" src="https://user-images.githubusercontent.com/304450/39247464-5f1162b4-4891-11e8-81ba-c16717676d10.png">